clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -331,7 +336,10 @@ def build_model(self,
             The Docker container registry to push the freshly built model to. Note
             that if you are running Clipper on Kubernetes, this registry must be accesible
             to the Kubernetes cluster in order to fetch the container from the registry.
-
+        batch_size : int, optional
+            The user defined batch size.
+            The default value is -1, which means that the user did not define a specific batch size.
```

Comment:
Update the documentation for this method in the same way as for `build_and_deploy_model`.